NEWS.md

```diff
@@ -1,3 +1,74 @@
+* Confdb: document confdb in snapctl help messages
+* Confdb: only confdb hooks wait if snaps are disabled
+* Confdb: relax confdb change conflict checks
+* Confdb: remove empty parent when removing last leaf
+* Confdb: support parsing field filters
+* Confdb: wrap confdb write values under "values" key
+* dm-verity for essential snaps: add new naming convention for verity files
+* dm-verity for essential snaps: add snap integrity discovery
+* dm-verity for essential snaps: fix verity salt calculation
+* Assertions: add hardware identity assertion
+* Assertions: add integrity stanza in snap resources revisions
+* Assertions: add request message assertion required for remote device management
+* Assertions: add response-message assertion for secure remote device management
+* Assertions: expose WithStackedBackstore in RODatabase
+* Packaging: cross-distro | install upstream NEWS file into relevant snapd package doc directory
+* Packaging: cross-distro | tweak how the blocks injecting $SNAP_MOUNT_DIR/bin are generated as required for openSUSE
+* Packaging: remove deprecated snap-gdb-shim and all references now that snap run --gdb is unsupported and replaced by --gdbserver
+* Preseed: call systemd-tmpfiles instead handle-writable-paths on uc26
+* Preseed: do not remove the /snap dir but rather all its contents during reset
+* snap-confine: attach name derived from security tag to BPF maps and programs
+* snap-confine: ensure permitted capabilities match expectation
+* snap-confine: fix cached snap-confine profile cleanup to report the correct error instead of masking backend setup failures
+* snap-confine: Improve validation of user controlled paths
+* snap-confine: tighten snap cgroup checks to ensure a snap cannot start another snap in the same cgroup, preventing incorrect device-filter installation
+* core-initrd: add 26.04 ubuntu-core-initramfs package
+* core-initrd: add missing order dependency for setting default system files
+* core-initrd: avoid scanning loop and mmc boot partitions as the boot disk won't be any of these
+* core-initrd: make cpio a Depends and remove from Build-Depends
+* core-initrd: start plymouth sooner and reload when gadget is available
+* Cross-distro: modify syscheck to account for differences in openSUSE 16.0+
+* Validation sets: use in-flight validation sets when calling 'snapctl install' from hook
+* Prompting: remove polkit authentication when modifying/deleting prompting rules
+* LP: #2127189 Prompting: do not record notices for unchanged rules on snapd startup
+* AppArmor: add free and pidof to the template
+* AppArmor: adjust interfaces/profiles to cope with coreutils paths
+* Interfaces: add support for compatibility expressions
+* Interfaces: camera | enable prompting
+* Interfaces: checkbox-support | complete overhaul
+* Interfaces: cuda-driver-libs, egl-driver-libs, gbm-driver-libs, opengl, opengl-driver-libs, opengles-driver-libs | allow access to nvidia graphics libraries exported by snaps
```

Comment:
Maybe:
* Interfaces: define vulkan-driver-libs, cuda-driver-libs, egl-driver-libs, gbm-driver-libs, opengl-driver-libs, and opengles-driver-libs
* Interfaces: allow snaps on classic access to nvidia graphics libraries exported by *-driver-libs interfaces